ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-gnmi.j2

```diff
@@ -5,14 +5,25 @@
 
 ### Management API GNMI Summary
 
-{%     if management_api_gnmi.enable_vrfs is defined %}
-| VRF with GNMI | OCTA |
-| ---------- | ---------- |
+| VRF | OCTA | Notification |
+| --- | ---- | ------------ |
+{%     if management_api_gnmi.octa is arista.avd.defined %}
```

Comment:
This has to be defined true to set it. Please fix this.

For example: 
```
management_api_gnmi:
  octa: false                              <---- This will enable the octa even though logically, we are trying to set it off.
```

Reply:
Needed change in both templates.

Reply:
I think I did it now, tested both `octa: false` or when `octa` flag is not set and works now